presentations/beman_overview_wg21_202406.org

```diff
@@ -47,16 +47,33 @@ Founded at C++Now in 2024 the project strives to aggregate libraries proposed fo
 - Co-founder / funder of C++-Now
 ** What's wrong with Boost?
 #+BEGIN_NOTES
-The short version: the world that Boost was born in no longer exists. In addition, Boost is no longer the source of the majority of library proposals. WG21 and users have lost something big - a single place to go see c++ next libraries.  Boost cannot be molded into what we need for a variety of reasons. Beman is a new effort (sponsored by the Boost Foundation) to provide that one place to go - using the latest tooling and best practices. Boost libraries are welcome in Beman as are any proposed for standardization.
+The short version: the world that Boost was born in no longer exists. In addition, Boost is no longer the source of the majority of library proposals. WG21 and users have lost something big - a single place to go see C++ next libraries.  Boost cannot be molded into what we need for a variety of reasons. Beman is a new effort (sponsored by the Boost Foundation) to provide that one place to go - using the latest tooling and best practices. Boost libraries are welcome in Beman as are any proposed for standardization.
 
 reviews are stuck in 1990 - should use tools!
 
 #+END_NOTES
+
+# DJS: "reviews are stuck in 1990 - should use tools!" could be interpreted as a
```

Comment:
removed -- reminder these are speakers notes not on screen.  Normally when I distribute the pdf of these they aren't in the content -- that said, it's good to have the right notes to stay on point.